predicators/envs/cover.py

```diff
@@ -86,6 +86,9 @@ def simulate(self, state: State, action: Action) -> State:
         if held_block is None and above_block is not None:
             grasp = pose - state.get(above_block, "pose")
             next_state.set(self._robot, "hand", pose)
+            if "hand_empty" in self._robot_type.feature_names:
```

Comment:
@tomsilver is this if stmt ok or is there a better way to do this?

Reply:
hmmm yeah this seems fine